source/geometry/FloatTransform3D.ooc

```diff
@@ -244,16 +244,16 @@ FloatTransform3D: cover {
 		This new(focalLength, 0, 0, 0, 0, focalLength, 0, 0, 0, 0, focalLength, 1.0f, 0, 0, 0, 0)
 	}
 	referenceToNormalized: func ~float (imageSize: FloatVector2D) -> This {
-		toReference := This createScaling(imageSize x / 2.0f, imageSize y / 2.0f, 1.0f)
-		toNormalized := This createScaling(2.0f / imageSize x, 2.0f / imageSize y, 1.0f)
+		toReference := This createScaling(imageSize x / 2.0f, imageSize y / 2.0f, imageSize x / 2.0f)
```

Comment:
I don't really know too much about what's going on, but I'm curious, why scale in `z` direction with `x`?